FlowCrypt/Models/Realm Models/KeyInfo.swift

```diff
@@ -13,12 +13,24 @@ enum KeySource: String {
 enum KeyInfoError: Error {
     case missedPrivateKey(String)
     case notEncrypted(String)
+    case missedKeyIds
 }
 
 final class KeyInfo: Object {
+    var primaryFingerprint: String? {
+        allFingerprints.first
+    }
```

Comment:
A KeyInfo object always must have a primaryFingerprint, so I would force this, or check for it and throw.

Reply:
actually this is already checked here 
```
guard keyDetails.ids.isNotEmpty else {
  assertionFailure("KeyDetails KeyIds should not be empty")
  throw KeyInfoError.missedKeyIds
}

...

self.allFingerprints.append(objectsIn: keyDetails.ids.map(\.fingerprint))
```